<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: &quot;#7C93C3&quot;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
      <widget class="QLabel" name="user_label">
       <property name="font">
        <font>
         <family>Tbilisi</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: &quot;#1E2A5E&quot;</string>
       </property>
       <property name="text">
        <string>მომხმარებლის სახელი</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" alignment="Qt::AlignLeft">
      <widget class="QLineEdit" name="password_input">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: &quot;#E1D7B7&quot;;
border: &quot;5px solid #E1D7B7&quot;;
color: &quot;#1E2A5E&quot;</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="login_button">
       <property name="font">
        <font>
         <family>Tbilisi</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: &quot;#55679C&quot;;
color: &quot;#E1D7B7&quot;</string>
       </property>
       <property name="text">
        <string>შესვლა</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignLeft">
      <widget class="QLineEdit" name="user_input">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: &quot;#E1D7B7&quot;;
border: &quot;5px solid #E1D7B7&quot;;
color: &quot;#1E2A5E&quot;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
      <widget class="QLabel" name="password_label">
       <property name="font">
        <font>
         <family>Tbilisi</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: &quot;#1E2A5E&quot;</string>
       </property>
       <property name="text">
        <string>პაროლი</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="title_label">
       <property name="font">
        <font>
         <family>Tbilisi</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: &quot;#1E2A5E&quot;</string>
       </property>
       <property name="text">
        <string>გაიარეთ ავტორიზაცია</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
